Renaming a file should re-add the extension if missing 11903 (#11908)
JabRef/jabref: CHANGELOG.md

- When fulltext search is selected but indexing is deactivated, a dialog is now shown asking if the user wants to enable indexing now [#9491](https://github.com/JabRef/jabref/issues/9491)
- We changed instances of 'Search Selected' to 'Search Pre-configured' in Web Search Preferences UI. [#11871](https://github.com/JabRef/jabref/pull/11871)
- We added a new CSS style class `main-table` for the main table. [#11881](https://github.com/JabRef/jabref/pull/11881)

> - Rename a file was changed, if there is no extension provided in the new name, the old one is used. [#11903](https://github.com/JabRef/jabref/issues/11903)

Reviewer: ```suggestion
- When renaming a file, the old extension is now used if there is none provided in the new name. [#11903](https://github.com/JabRef/jabref/issues/11903)
```
